test/metabase/api/collection_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns metabase.api.collection-test
   "Tests for /api/collection endpoints."
-  (:require [clojure.string :as str]
+  (:require [clj-time.core :as time]
```

Comment:
Use `java-time` (wraps the Java 8 `java.time` libraries) instead of `clj-time`. [`clj-time` is deprecated](https://github.com/clj-time/clj-time#clj-time)

We have tons of tooling around `java.time` too such as `metabase.util.date2` and the reader literal `#t` for any sort of ISO-8601 date/time. Plus the classes are all integrated with JDBC (some out of the box, others I did manually) so you can use them directly in query.

Don't use Joda/`clj-time.core` or `java.util.Date`/`java.sql.Timestamp`/etc. unless you absolutely have to for external library interop (example -- some JDBC drivers don't support the `java.time` classes yet) -- and even then you should use `java.time` until the very last minute. `java.util.Date` has caused a crazy amount of timezone bugs

Reply:
I had forgotten i left this in there. you can't pass in a timestamp on the revision (it gets `:%now` unconditionally assoced onto it). I forgot which wrapper was correct and just did `(apropos "minus")` and saw that pop up. Below i use java.time interop to actually set the time back on a revision so they have a well-defined order under last-edit.